src/mbgl/renderer/buckets/fill_extrusion_bucket.cpp

```diff
@@ -109,7 +109,7 @@ void FillExtrusionBucket::addFeature(const GeometryTileFeature& feature, const G
 
                     const Point<double> perp = util::unit(util::perp(d1 - d2));
                     const auto dist = util::dist<int16_t>(d1, d2);
-                    if (edgeDistance + dist > std::numeric_limits<int16_t>::max()) {
+                    if (edgeDistance + dist > std::numeric_limits<std::size_t>::max()) {
```

Comment:
always false?

Reply:
I've updated my change. Instead of changing the template argument, now I'm casting the limits result to std::size_t. This way, the semantic is kept.